couchbase/src/api/buckets.rs

```diff
@@ -177,10 +177,10 @@ pub struct BucketSettingsBuilder {
 }
 
 impl BucketSettingsBuilder {
-    pub fn new<S: Into<String>>(name: S, ram_quota_mb: u64) -> BucketSettingsBuilder {
+    pub fn new<S: Into<String>>(name: S) -> BucketSettingsBuilder {
         Self {
             name: name.into(),
-            ram_quota_mb,
+            ram_quota_mb: 0,
```

Comment:
should we set the default to 100 for simplicity?

Reply:
Yeah I like that idea